kinetic/distribution.yaml

```diff
@@ -14911,17 +14911,17 @@ repositories:
     doc:
       type: git
       url: https://github.com/mgruhler/soem.git
-      version: master
+      version: kinetic 
```

Comment:
A little hard to see in the test output, but you have a trailing whitespace here and below in the source.

Reply:
Thank you so much.
Silly me. How did I not get that???

I updated the branch just now. Thanks